adoptopenjdk-api-v3-persistance/src/main/kotlin/net/adoptopenjdk/api/v3/DownloadStatsInterface.kt

```diff
@@ -52,27 +57,67 @@ class DownloadStatsInterface(
 
         // look for stats up to 10 days before the requested start, this is because for a daily diff we need the
         // previous entry before the period started
-        val stats = getStats(periodStart.minusDays(10), periodEnd, featureVersion, dockerRepo, statsSource)
+        val stats = getStats(periodStart.minusDays(10), periodEnd, featureVersion, dockerRepo, jvmImpl, statsSource)
 
         return calculateDailyDiff(stats, periodStart, periodEnd, days)
     }
 
+    suspend fun getMonthlyTrackingStats(
+        to: ZonedDateTime? = null,
+        source: StatsSource? = null,
+        featureVersion: Int? = null,
+        dockerRepo: String? = null,
+        jvmImpl: JvmImpl? = null
+    ): List<MonthlyDownloadDiff> {
+
+        val periodEnd = to ?: TimeSource.now()
```

Comment:
Would this mean that the last entry will always be a partial of the month? It may be better to report up to the end of the last month otherwise the current month will increase day by day

Reply:
Yes, I thought it would be nice to show the current month and didn't see much of a problem showing a partial result as anyone reading it should be able to figure out that obviously the current month would be lower, but suppose that's up to you.

Reply:
I think we should probably exclude the current month, I dont think it would be what people expect

Reply:
Done